plugin-repos/starlark-apps/tronbyte_repository.py

```
@@ -0,0 +1,472 @@
+            """Fetch a single app's manifest (runs in thread pool)."""
+            app_id = app_info['id']
+            manifest_path = f"{self.APPS_PATH}/{app_id}/manifest.yaml"
+            content = self._fetch_raw_file(manifest_path)
+            if content:
+                try:
+                    metadata = yaml.safe_load(content)
+                    if not isinstance(metadata, dict):
+                        metadata = {}
+                    metadata['id'] = app_id
+                    metadata['repository_path'] = app_info.get('path', '')
+                    return metadata
```

Ruff (0.15.1)

[warning] Consider moving this statement to an `else` block

(TRY300)